packages/services/src/kernel/future.ts

```diff
@@ -221,15 +203,19 @@ export class KernelFutureHandler<
     }
   }
 
-  private async _handleStdin(msg: KernelMessage.IStdinMessage): Promise<void> {
+  protected async _handleStdin(
```

Comment:
our convention is that only private functions start with `_`, not protected functions